### ISSUE TITLE
FPS Controller + Collision

### ISSUE THREAD

**Issue by VerdeCreeper** [Calinou/godot-sponza#78]
Hi, is it possible to make a new Sponza scene but instead of being free camera can be a first person character, alongside with collision? Or just tell me how to import the model to other projects/scenes with a compatible script

**Comment by Calinou** [comment]
Yes, but I don't intend to add a FPS controller to this demo. I can't provide support as to how you'd fit a FPS controller in this demo, so you'll have to find tutorials about creating a FPS controller in Godot elsewhere.

**Comment by VerdeCreeper** [comment]
Alright, but atleast tell me how to give collision to the model and use it in another scenes
